test/functional/info/info-basic.bats

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup() {
+
+	create_test_environment "$TEST_NAME" 10 1
+
+}
+
+test_teardown() {
+
+	destroy_test_environment "$TEST_NAME"
+
+}
+
+
+@test "INF001: Basic test, Check info verbose" {
+
+	run sudo sh -c "$SWUPD info $SWUPD_OPTS --verbose"
+	assert_status_is "$SWUPD_OK"
+	expected_output=$(cat <<-EOM
+		Distribution:      Swupd Test Distro
+		Installed version: 10 \(format 1\)
+		Version URL:       .*web-dir
+		Content URL:       .*web-dir
+	EOM
+	)
+
+	assert_regex_is_output "$expected_output"
```

Comment:
avoid using regex to check output when possible, in this case you can just do:
```
	expected_output=$(cat <<-EOM
		Distribution:      Swupd Test Distro
		Installed version: 10 (format 1)
		Version URL:       file://$TEST_DIRNAME/web-dir
		Content URL:       file://$TEST_DIRNAME/web-dir
	EOM
	)

	assert_is_output "$expected_output"
```
I think it looks much cleaner and provides better output when there are failures.

Reply:
Ya agreed. Thanks will change